mypy.ini

```diff
@@ -5,4 +5,5 @@ check_untyped_defs = True
 warn_redundant_casts = True
 warn_unused_ignores = True
 allow_untyped_defs = False
-strict_equality = True
\ No newline at end of file
+strict_equality = True
+ignore_missing_imports = True
```

Comment:
For imported python packages that don't have types associated (feature gate package for example)

Reply:
i think we should try to suppress it first at that one point then with `# type: ignore`

Reply:
how is this resolved?

Reply:
I'd resolved it after answering the initial question. I've unresolved it now

Reply:
`ignore_missing_imports` is an okay setting for specific packages, i.e. like what was in here before for pyteal etc. I wouldn't advise applying it globally like this no.

I would also suggest adding `PGH003` to the `[tool.ruff]` `select` list, `# type: ignore` is too broad. I can see why you've needed to do it due to changes in `algosdk` (it would be better if `algosdk` types were actually reflective of the response and not just always a union of all possible response types for all requests). You can also narrow types by using an `assert isinstance(response, dict)`, which means we don't hide other issues. You might even be able to ignore type union issues for all of `algosdk`?

Reply:
went thru and removed the ignores i added, removed the ignore_missing_imports